DemoApp/DemoApp/Views/TMCardPage.xaml

```diff
@@ -92,7 +92,7 @@
           <StackLayout BackgroundColor="Transparent"
                        HorizontalOptions="End"
                        Padding="0,10,0,10">
-            <local:TMButton Title="Button"></local:TMButton>
+            <local:TMButton Title="Button" Size="Default"></local:TMButton>

```

Comment:
Why default added here?

Reply:
If we don't set size "default" ,size is  not set by default .

Reply:
I think we have to set it in TMButton.xaml.cs size value as default

Reply:
Done